lib/DateFilter/DateFilter.js

```diff
@@ -52,18 +52,22 @@ const DateFilter = ({ activeFilters, filterHandlers, name }) => {
       separator={false}
     >
       <DateFilterForm
+        allowNull={allowNull}
```

Comment:
https://github.com/folio-org/ui-agreements/pull/739#discussion_r563998922

Reply:
Change made, to `showNoDateCheckbox` (switched from showNoEndDateCheckbox since this component is covers both start and end dates)